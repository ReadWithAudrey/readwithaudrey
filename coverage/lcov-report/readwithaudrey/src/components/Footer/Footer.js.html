<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for readwithaudrey/src/components/Footer/Footer.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../../index.html">All files</a> / <a href="index.html">readwithaudrey/src/components/Footer</a> Footer.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>2/2</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>1/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>2/2</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">14x</span>
<span class="cline-any cline-yes">9x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React from 'react'
import { Link } from 'gatsby'
import { StyledFooter, FooterNav, Copyright } from './Footer.style'
&nbsp;
const Footer = () =&gt; (
  &lt;StyledFooter&gt;
    &lt;FooterNav&gt;
      &lt;Link to="/" className="no-underline white ph1"&gt;
        Home
      &lt;/Link&gt;
      &lt;Link to="/story/" className="no-underline white ph1"&gt;
        Our Story
      &lt;/Link&gt;
      &lt;Link to="/faq/" className="no-underline white ph1"&gt;
        FAQs
      &lt;/Link&gt;
      &lt;Link to="/contact/" className="no-underline white ph1"&gt;
        Contact Us
      &lt;/Link&gt;
    &lt;/FooterNav&gt;
    &lt;Copyright&gt;
      &amp;copy; Audrey
      &lt;Link to="/Terms/" className="no-underline black ph1"&gt;
        Terms
      &lt;/Link&gt;
      &lt;Link to="/Privacy/" className="no-underline black ph1"&gt;
        Privacy
      &lt;/Link&gt;
    &lt;/Copyright&gt;
  &lt;/StyledFooter&gt;
)
&nbsp;
export default Footer
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Fri Oct 12 2018 12:51:26 GMT+0100 (British Summer Time)
</div>
</div>
<script src="../../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../../sorter.js"></script>
</body>
</html>
