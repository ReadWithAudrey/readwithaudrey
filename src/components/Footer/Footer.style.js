import styled from 'styled-components'

export const StyledFooter = styled.footer`
  background: #fff;
`

export const FooterNav = styled.span`
  background: #fff;
`

export const Copyright = styled.span`
  background: #fff;
  color: #000;
  width: 800px;
  margin: 0 auto;
`
