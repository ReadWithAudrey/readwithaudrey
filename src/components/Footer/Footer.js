import React from 'react'
import { Link } from 'gatsby'
import { StyledFooter, FooterNav, Copyright } from './Footer.style'

const Footer = () => (
  <StyledFooter>
    <FooterNav>
      <Link to="/" className="no-underline white ph1 ">
        Home
      </Link>
      <Link to="/Story/" className="no-underline white ph1">
        Our Story
      </Link>
      <Link to="/faq/" className="no-underline white ph1">
        FAQs
      </Link>
      <Link to="/contact/" className="no-underline white ph1">
        Contact Us
      </Link>
    </FooterNav>
    <Copyright>
      &copy; Audrey
      <Link to="/Terms/" className="no-underline black ph1">
        Terms
      </Link>
      <Link to="/Privacy/" className="no-underline black ph1">
        Privacy
      </Link>
    </Copyright>
  </StyledFooter>
)

export default Footer
