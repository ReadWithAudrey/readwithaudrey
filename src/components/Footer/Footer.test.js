import React from 'react'
import renderer from 'react-test-renderer'
import Footer from './Footer'

describe('Footer component snapshot test', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<Footer />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
