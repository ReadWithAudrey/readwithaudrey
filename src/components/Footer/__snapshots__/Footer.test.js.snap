// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component snapshot test renders correctly 1`] = `
<footer
  className="sc-bdVaJa flex flex-column justify-center items-center montserrat bg-green w-100 fePDys"
>
  <span
    className="sc-bwzfXH flex items-center justify-around bg-green w-100 jWjOZT"
  >
    <Link
      className="no-underline white ph1 "
      to="/"
    >
      Home
    </Link>
    <Link
      className="no-underline white ph1"
      to="/story/"
    >
      Our Story
    </Link>
    <Link
      className="no-underline white ph1"
      to="/faq/"
    >
      FAQs
    </Link>
    <Link
      className="no-underline white ph1"
      to="/contact/"
    >
      Contact Us
    </Link>
  </span>
  <span
    className="sc-htpNat flex justify-around items-center bg-white black w-100 jbutSu"
  >
    Â© Audrey
    <Link
      className="no-underline black ph1"
      to="/Terms/"
    >
      Terms
    </Link>
    <Link
      className="no-underline black ph1"
      to="/Privacy/"
    >
      Privacy
    </Link>
  </span>
</footer>
`;
